ui/perfherder/alerts/AlertStatusCountdown.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClock } from '@fortawesome/free-regular-svg-icons';
+import { faCheck } from '@fortawesome/free-solid-svg-icons';
+
+import SimpleTooltip from '../../shared/SimpleTooltip';
+import {
+  getStatus,
+  getTimeDifference,
+  getCountdownText,
+} from '../perf-helpers/helpers';
+import { weekdays } from '../perf-helpers/constants';
+
+export default class AlertStatusCountdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+
+    this.isWeekend = false;
+    this.showCountdownToTriageIcon = true;
+  }
+
+  getDueDateCountdownsStatus() {
+    const { alertSummary } = this.props;
+
+    const now = new Date(Date.now());
+    const currentDay = now.getDay();
+    const triageDueDate = new Date(alertSummary.triage_due_date);
```

Comment:
You can use destructuring here and use camel case to keep the naming convention enforced by JavaScript.

Reply:
```
    let { triageDueDate } = alertSummary;
    triageDueDate = new Date(triageDueDate);
```
like this?

Reply:
let { triageDueDate: triage_due_date } = alertSummary;
    triageDueDate = new Date(triageDueDate);